play/type-shooter/js/app.js

```diff
@@ -34,18 +47,66 @@ const STARTING_LIVES = 10;
 (function initUI() {
     // show instructions modal
     $("#myModal").modal("show");
+
+    // keep page scrolled to the very top
+    document.querySelector("body").scrollTop = 0;
+
+    // edit styles for balloonsPopped text and livesLeft text
+    let livesLeft = document.getElementById("livesLeft");
+    livesLeft.style.fontSize = "25px";
+    livesLeft.style.fontWeight = "bold";
+    livesLeft.style.webkitTextStrokeWidth = "1px";
+    livesLeft.style.webkitTextStrokeColor = "black";
+    livesLeft.style.webkitTextFillColor = "rgb(0, " + greenTextColor + ", 0)";
+
+    let poppedBalloons = document.getElementById("balloonsPopped");
+    poppedBalloons.style.fontSize = "25px";
+    poppedBalloons.style.fontWeight = "bold";
+    poppedBalloons.style.webkitTextStrokeWidth = "1px";
+    poppedBalloons.style.webkitTextStrokeColor = "black";
+    poppedBalloons.style.webkitTextFillColor = "rgb(255, 255, 255)";
+
+    let timeSurvived = document.querySelector("#timeSurvived");
+    timeSurvived.style.fontSize = "25px";
+    timeSurvived.style.fontWeight = "bold";
+    timeSurvived.style.webkitTextStrokeWidth = "1px";
+    timeSurvived.style.webkitTextStrokeColor = "black";
+    timeSurvived.style.webkitTextFillColor = "rgb(255, 255, 255)";
+
+    let input = document.getElementById("input");
+    input.style.fontSize = "25px";
+    input.style.fontWeight = "bold";
+    input.style.webkitTextStrokeWidth = "1px";
+    input.style.webkitTextStrokeColor = "black";
+    input.style.webkitTextFillColor = "rgb(255, 255, 255)";
+    input.size = "" + HARD_LENGTH;
+    input.maxLength = "" + HARD_LENGTH;
 })();
 
 /**
  * Starts the Type Shooter game loop
  */
-function startGame() {
+async function startGame() {
 
+    // wait to finish reading in the text file
+    let promise = await readTextFile(WORD_LIST);
+
+    // let the user know if an error occurred
+    if (promise == -1) {console.log("Error: Input text file could not be read");}
```

Comment:
If you're going to have it on one line with only one statement in the if statement, you should just get rid of the brackets.